content/blog/chess-knight-problem/index.md

```diff
@@ -0,0 +1,113 @@
+---
+{
+  title: "Chess Knight Problem: a quick and dirty solution in JavaScript",
+  description: "Here I present a quick and dirty solution to a common interview question where the solution is not nearly as complex as it may first appear.",
+  published: "2020-04-29T12:27:06.284Z",
+  authors: ["thodges314"],
+  tags: ["javascript", "chess", "knight", "computer science", "interview"],
```

Comment:
```suggestion
  tags: ["javascript", "computer science", "interview"],
```

Let's remove the "Chess" and "Knight" tags. They make the tags a bit long in the card view

![Card view showing all the tags and overflowing](https://user-images.githubusercontent.com/9100169/80640745-b5abba00-8a18-11ea-8d4f-69dcc8b64cc0.png)